### ISSUE TITLE
Algorithm generates unrealistically long or short activities

### ISSUE THREAD

**Issue by dhixsingh** [matsim-melbourne/demand#2]
Pulled out from #1. 

While the plan generation algorithm does a reasonable job of matching activity start/end times to input VISTA distributions, it does not match activity durations. Due to this one can get unrealistically long or short activities. 

Some incomplete logic is here:
https://github.com/matsim-melbourne/demand/blob/266d4811c8b25a3eff1094e74752401c4819765d/R/plan.R#L92-L100

**Comment by dhixsingh** [comment]
As of commit 9b1cfd6, an activity duration graph is also output with the VISTA-like plan generation step. Below is what the produced durations look like with respect to the expected distributions from VISTA (for a run producing 5k plans). This is already fairly acceptable as is, so I will not look to improve it any further.

<img width="880" alt="Screen Shot 2020-10-28 at 9 48 49 am" src="https://user-images.githubusercontent.com/7893676/97370716-b6f46b80-1903-11eb-904f-456520b272ab.png">

I also looked further into the reported seemingly unrealistic long durations for certain activities, for instance **Pickup**, and found that that is what it is in the VISTA data. Below are the top 20 VISTA persons with the longest pickup/dropoffs (in mins). Overall, accounting for weights, this is a small percentage of the population.

<img width="864" alt="Screen Shot 2020-10-28 at 9 44 45 am" src="https://user-images.githubusercontent.com/7893676/97371077-8eb93c80-1904-11eb-90d6-526791d5ea48.png">

Below are the top 25 plans generated (0.5% of the total 5k) with the longest durations (1 bin is 30mins). These reflect the above VISTA persons. The remaining 99.5% of the pickup/dropoffs are less than 3.5hrs long, which again seems reasonable.

<img width="434" alt="Screen Shot 2020-10-28 at 9 48 25 am" src="https://user-images.githubusercontent.com/7893676/97371488-6847d100-1905-11eb-90af-95d7c8c50353.png">

In summary, the current activity duration generation seems fairly reasonable as is. No adjustment to the algorithm seems necessary here, so am closing this for now.

/cc @jafshin
